bump version for utility-connect 2.0 (#9)
ArcadiaPower/utility-connect-react: README.md

Callback functions triggered at key points in the Utility Connect flow. Expects an object with key/value pairs where keys are as documented below.

**`onClose`**: callback function that is triggered when the Utility Connect component is closed.

> **`onError`**: callback function that is triggered when an error occurs within the application.

Reviewer: Maybe something like:

```suggestion
**`onError`**: callback function that is triggered when an error occurs within the Utility Connect process.
```

**`onOpen`**: callback function that is triggered when the Utility Connect component is opened.

**`onSuccess`**: callback function that is triggered when utility credentials are successfully verified.

> **`onTimeout`**: callback function that is triggered when the Utility Connect component timesout while trying to validate utility credentials.

Reviewer: ```suggestion
**`onTimeout`**: callback function that is triggered when the Utility Connect component times out awaiting verification of the utility credentials.
```

> **`onTimeout`**: callback function that is triggered when the Utility Connect component timesout while trying to validate utility credentials.

Reviewer 2: These should contain method signatures and payload specifications, eg

```
**`onSuccess(credentialsCreatedResponse)`** callback function that is triggered...

`credentialsCreatedResponse` is an object like:

Key | Type | Desc
--- | --- | ---
`data` | Object | Credentials-created API response
`data.user_id` | Number | User ID of created member
`data.whatever` | Blob | Zippity doo bop
```

```javascript
{
  onClose: () => { ... },

>   onError: () => { ... },

Reviewer: ```suggestion
  onError: ({ error }) => { ... },
```
